testsuite/performance/docker-compose-cluster.yml

```diff
@@ -15,7 +15,8 @@ services:
     mariadb:
         build: db/mariadb
         image: keycloak_test_mariadb:${KEYCLOAK_VERSION:-latest}
-        cpus: 1
+        cpuset: ${DB_CPUSET:-1}
```

Comment:
Does this parametrization actually work? I've tried it with the `cpus` option earlier and it didn't work.

Reply:
Yes it does.